File: Write_Program_in_any_lang/c_program/Vowel.c
#include <stdio.h>
int main() {
    char c;
    int lowercase_vowel, uppercase_vowel;
    printf("Enter an alphabet: ");
    scanf("%c", &c);
    lowercase_vowel = (c == 'a' || c == 'e' || c == 'i' || c == 'o' || c == 'u');
    uppercase_vowel = (c == 'A' || c == 'E' || c == 'I' || c == 'O' || c == 'U');

    if (lowercase_vowel || uppercase_vowel)
        printf("%c is a vowel.", c);
    else
        printf("%c is a consonant.", c);
    return 0;
}

File: Write_Program_in_any_lang/python/vowel.py
def isVowel(char):
	char=char.lower()
	vowel = ['a','e','i','o','u']
	if char in vowel:
		return True
	else:
		return False
while True:
	char = input('Enter any alphabet : ')
	if len(char)==1:
		break
	else:
		print('Please Enter Character of length 1')
if isVowel(char):
	print("Character '",char,"' is a vowel!")
else:
	print("Character '",char,"' is a consonant!")